pkg/registry/storage/volumeattachment/strategy.go

```diff
@@ -53,13 +55,20 @@ func (volumeAttachmentStrategy) PrepareForCreate(ctx context.Context, obj runtim
 		groupVersion = schema.GroupVersion{Group: requestInfo.APIGroup, Version: requestInfo.APIVersion}
 	}
 
+	volumeAttachment := obj.(*storage.VolumeAttachment)
+
 	switch groupVersion {
 	case storageapiv1beta1.SchemeGroupVersion:
 		// allow modification of status for v1beta1
 	default:
 		volumeAttachment := obj.(*storage.VolumeAttachment)
```

Comment:
This line can be removed.

The assignment happens on line 58

Reply:
Hi @tedyu, thank you for the suggestions. This PR has already merged. Could you please submit a cleanup PR for the nit you mentioned

Reply:
see #78659